Misc/NEWS.d/next/Library/2021-04-12-00-00-00.bpo-43776.p14y7a.rst

```diff
@@ -0,0 +1 @@
+Fix issue with :class:`subprocess.Popen` repr when args are provided as a string.
```

Comment:
"Fix issue" isn't specific.  Describe the behavior change so people can understand what is really new without going and opening the bug.  Wording more like this is nicer for the reader:

"When :class:`subprocess.Popen` args are provided as a string for ``shell=True`` use, the Popen instance repr now shows the right thing."